Tasks/SonarQubePostTest/PRCA/ReportProcessor.ps1

```diff
@@ -2,6 +2,80 @@ $ProjectGuidAndFilePathMap = @{}
 $ComponentKeyAndPathMap = @{}
 $ComponentKeyAndRelativePathCache = @{}
 
+
+#region Public
```

Comment:
The file contains a single "public" method, although PS does not provide a function visibility mechanism. It does provide such a mechanism should we choose to organize the scripts in a module, but I do not see enough value in doing so (modules are mostly for distribution)